resources/assets/components/Empty/empty.scss

```diff
@@ -0,0 +1,19 @@
+@import '~@dosomething/forge/scss/toolkit';
+
+.empty {
+  text-align: center;
+
+  .image {
```

Comment:
It's probably a good idea to scope this `.empty__image` so we don't accidentally have a naming conflict if we were ever to add an `.image` class elsewhere.